features/pull_request.feature

```diff
@@ -160,6 +160,41 @@ Feature: hub pull-request
     When I successfully run `hub pull-request`
     Then the output should contain exactly "the://url\n"
 
+  Scenario: Single-commit pull request with "--no-edit"
+    Given the GitHub API server:
+      """
+      post('/repos/mislav/coral/pulls') {
+        assert :title => 'Commit message 1',
+               :body => nil
+        status 201
+        json :html_url => "the://url"
+      }
+      """
+    Given I am on the "master" branch pushed to "origin/master"
+    When I successfully run `git checkout --quiet -b topic`
+    Given I make a commit with message "Commit message 1"
+    And the "topic" branch is pushed to "origin/topic"
+    When I successfully run `hub pull-request --no-edit`
+    Then the output should contain exactly "the://url\n"
+
+  Scenario: Multiple-commit pull request with "--no-edit"
+    Given the GitHub API server:
+      """
+      post('/repos/mislav/coral/pulls') {
+        assert :title => 'Commit message 1',
+               :body => nil
+        status 201
+        json :html_url => "the://url"
+      }
+      """
+    Given I am on the "master" branch pushed to "origin/master"
+    When I successfully run `git checkout --quiet -b topic`
+    Given I make a commit with message "Commit message 1"
+    Given I make a commit with message "Commit message 2"
+    And the "topic" branch is pushed to "origin/topic"
```

Comment:
Can there be an additional scenario that verifies that this works with `--push` as well?